package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "codejam",
+  "version": "1.0.0",
+  "eslintConfig": {
```

Comment:
You don't need `eslintConfig` property in _packages.json_ if you have `.eslintrc` file with the same rules.
And you have one😊